daemon/cluster/services.go

```diff
@@ -43,18 +43,21 @@ func (c *Cluster) GetServices(options apitypes.ServiceListOptions) ([]types.Serv
 	// be good to have accepted file check in the same file as
 	// the filter processing (in the for loop below).
 	accepted := map[string]bool{
-		"name":  true,
-		"id":    true,
-		"label": true,
-		"mode":  true,
+		"name":     true,
+		"id":       true,
+		"label":    true,
+		"mode":     true,
+		"runtimes": true,
```

Comment:
I was matching what was in the swarm `ListServicesRequest_Filters` but it probably makes more sense singular.